### DISCUSSION TITLE
add state of creation to filetag

### DISCUSSION THREAD

**Discussion by ghost** [DeaDBeeF-Player/deadbeef#2535] [Q&A]
i would like to setup a short command in deadbeef, which fetches the state of creation of a file so to store it in a new tag field.

under fb2k i used a little script with the formatting: %last_modified% and the destination field DATE CREATED, which would then contain something like: 2021-02-15 13:55:14

could you please give me a tip on how to achieve the same with deadbeef's shell commands under linux?

**Comment by Oleksiy-Yakovenko** [comment]
Deadbeef doesn't have functionality to fill the tags using title formatting scripts.

**Reply by ghost** [reply]
ok, i understand. thanks for letting me know. 

if there is nothing speaking against it, how about turning this question/missing functionality into a feature request?

**Reply by Oleksiy-Yakovenko** [reply]
github doesn't allow to move back from discussions into issues, so I created a new one
https://github.com/DeaDBeeF-Player/deadbeef/issues/2537
